contracts/ERC20Mixer.sol

```diff
@@ -26,20 +26,20 @@ contract ERC20Mixer is Mixer {
     token = _token;
   }
 
-  function _processDeposit() internal {
+  function _processDeposit() internal nonReentrant {
```

Comment:
This function itself does not update contract's state, so there is no obvious reason to protect it from reentrancy.  Consider moving “nonReentrant” modifier to the calling function that actually ought to be protected.

Reply:
Agree